app/views/console/comps/header.phtml

```diff
@@ -211,7 +211,7 @@
             required
             maxlength="36"
             class=""
-            pattern="^[a-zA-Z0-9][a-zA-Z0-9_.-]{1,36}$"
+            pattern="^[a-zA-Z0-9][a-zA-Z0-9._-]{0,36}$"
```

Comment:
```suggestion
            pattern="^[a-zA-Z0-9][a-zA-Z0-9._-]{0,35}$"
```

It needs to be 0,35. Just to make sure - even though we have a maxlength.

Reply:
Applied.